libcore/AbstractSlot.vala

```diff
@@ -94,6 +94,7 @@ namespace GOF {
         public abstract void initialize_directory ();
         public abstract unowned GLib.List<GOF.File>? get_selected_files ();
         public abstract void set_active_state (bool set_active, bool animate = true);
+        public abstract bool get_active_state ();
```

Comment:
Why not expose is_active property instead?

Reply:
Because the existing set function for that property requires two parameters, so it seems more consistent to provide a matching get function.

Reply:
Yeah, this makes sense